### PR TITLE
Minor

### PR DESCRIPTION
**Pull request by Crocodeele**
@kovalevvlad

### CODE REVIEW

File: graphics.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+                else:
+                    self.surface.blit(options[num][0], text_place[num])
+
+def get_events():
+    was_closed = False
+    was_clicked = False
+    mouse_pos = pygame.mouse.get_pos()
+
+    for event in pygame.event.get():
+        if event.type == pygame.QUIT:
```
Comment (kovalevvlad) [inline graphics.py]: `if event.type in (pygame.QUIT, pygame.KEYDOWN)`?

File: graphics.py [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+            was_closed = True
+        elif event.type == pygame.KEYDOWN:
+            was_closed = True
+        elif event.type == pygame.MOUSEBUTTONDOWN:
+            was_clicked = True
+
+    return {"was_closed": was_closed,
```
Comment (kovalevvlad) [inline graphics.py]: try using named tuples

File: graphics.py [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+            "was_clicked": was_clicked,
+            "mouse_pos": mouse_pos}
+
+
+def undraw(game_window, planet):
+    pygame.draw.circle(game_window.surface, (0, 0, 0), (int(planet.x), int(planet.y)), int(planet.size))
```
Comment (kovalevvlad) [inline graphics.py]: why do you need to convert to int?

File: main.py [hunk 4]
```diff
@@ -1 +1,186 @@
...
+from os import path
+from random import randint
 
+# game planets data
+planets = []
+
+
+def check_for_collision(sounds, window):
+    def check_is_in_screen(pl1, window):
+        if (pl1.x - pl1.size > window.size_x) or (pl1.y - pl1.size > window.size_y) or \
```
Comment (kovalevvlad) [inline main.py]: get rid of the if:
`return (pl1.x - pl1.size > window.size_x) or (pl1.y - pl1.size > window.size_y) or (pl1.x + pl1.size < 0) or (pl1.y + pl1.size < 0)`

Comment (kovalevvlad) [inline main.py]: Also I don't really understand the algorithm, can you explain? I would expect to see something like:
`(0 < point.x < window.width) and (0 < point.y < window.height)`

File: physics.py [hunk 5]
```diff
@@ -0,0 +1,68 @@
+import math
+import pygame
+
+mass_to_size_koeff = 0.4
```
Comment (kovalevvlad) [inline physics.py]: coeff*

File: main.py [hunk 6]
```diff
@@ -1 +1,186 @@
...
+                (pl1.x + pl1.size < 0) or (pl1.y + pl1.size < 0):
+            return True
+        else:
+            return False
+
+    # this is used to avoid errors after removing a planet
+    was_changed = False
+
+    for y in range(0, len(planets)):
```
Comment (kovalevvlad) [inline main.py]: just iterate over planets? If you need index use enumerate

File: main.py [hunk 7]
```diff
@@ -1 +1,186 @@
...
+                planets.pop(y)
+                was_changed = True
+
+
+def attract_once():
+    for y in range(0, len(planets)):
+        for x in range(y, len(planets)):
+            if not x == y:
+                #calculating gravity force and then adding it to one planet
+                force = phsx.gravity_force(planets[y], planets[x])
```
Comment (kovalevvlad) [inline main.py]: don't rename imports unless it's a convention e.g. pandas as pd or numpy as np etc.

File: main.py [hunk 8]
```diff
@@ -1 +1,186 @@
...
+    elif side == 3:
+        x = randint(100, window.size_x - 100)
+        y = 0
+        dx = randint(-5, 5) * size / 3
+        dy = randint(1, 5) * size / 3
+    elif side == 4:
+        x = randint(100, window.size_x - 100)
+        y = window.size_y
+        dx = randint(-5, 5) * size / 3
+        dy = randint(-5, -1) * size / 3
```
Comment (kovalevvlad) [inline main.py]: 20 lines of duplication above

File: main.py (hunk 4 above)
Comment (Crocodeele) [inline main.py]: That would make the circle disappear when it's **center** is off screen, my implementation waits until it's fully off screen, and then removes the circle.

File: main.py (hunk 8 above)
Comment (Crocodeele) [inline main.py]: each one is slightly different because

File: graphics.py (hunk 3 above)
Comment (Crocodeele) [inline graphics.py]: change in x and y is a float, therefore the position is a float, but pygame accepts integer only coordinates

**Comment by Crocodeele** [comment]
added tests @kovalevvlad

**Pull request merged**